src/components/Book.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+const Book = ({ title, author }) => (
+  <>
```

Comment:
- [ ] Please, could you describe from which component these props are coming? Please, consider fixing this issue, to make your app work correctly, as required 👍